src/main/java/io/rpg/viewmodel/LocationViewModel.java

```diff
@@ -45,4 +47,8 @@ public void initialize(URL location, ResourceBundle resources) {
       mapImageView.setFitHeight(newImg.getHeight());
     });
   }
+
+  public void addChild(ImageView child){
+    parent.getChildren().add(child);
+  }
```

Comment:
```suggestion
  public void addChild(ImageView child) {
    parent.getChildren().add(child);
  }
```

Dokładnie o to chodziło, gicik.